test/TestProjects/MultipleInputFiles/Generated/MultipleInputFilesRestClient.cs

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using MultipleInputFiles.Models;
+
+namespace MultipleInputFiles
+{
+    internal partial class MultipleInputFilesRestClient
+    {
+        private Source? source;
+        private ClientDiagnostics _clientDiagnostics;
+        private HttpPipeline _pipeline;
+
+        /// <summary> Initializes a new instance of MultipleInputFilesRestClient. </summary>
+        /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
+        /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
+        /// <param name="source"> source - server parameter. </param>
+        public MultipleInputFilesRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, Source? source = default)
+        {
+            this.source = source ?? new Source("value1");
```

Comment:
It probably would be better to make `source` non-nullable, but it would require to add fields to RestClient and use them in all writers.